doltpy/cli/read/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .read import read_rows, read_columns, read_pandas, read_rows_sql, read_columns_sql, read_pandas_sql
```

Comment:
i like implicit namespacing references, but in most of the code you use explicit

Reply:
What do you mean by that specifically? You prefer the style above to be used in the code as well as the package namespace definitons?

Reply:
you should pick one at stick to it? PEP is indecisive about this, either works with Python3

Reply:
Agreed. Moving everything to implicit, as it's more succinct.